client/Components/App.jsx

```diff
@@ -51,4 +50,4 @@ class App extends Component {
   }
 }
 
-ReactDOM.render(<App />, document.getElementById("root"));
```

Comment:
did you remove this so that you can run tests? I had an issue with running tests on App